third_party/silabs/efr32_sdk.gni

```diff
@@ -54,6 +54,13 @@ declare_args() {
   # Argument to enable IPv4 for wifi
   # aligning to match chip_inet_config_enable_ipv4 default configuration
   chip_enable_wifi_ipv4 = false
+  silabs_gen_folder = ""
+
+  # Calls SLC-CLI at run time
+  slc_generate = false
```

Comment:
Unfortunately no. SLC requires a python venv and since the efr32.py already run in a python venv it wouldn't be able to call slc (already tried). There seems to be a limitation with nested python virtual environment. :(